src/Filters/coefficients.jl

```diff
@@ -44,12 +44,16 @@ Base.promote_rule(::Type{ZeroPoleGain{D,Z1,P1,K1}}, ::Type{ZeroPoleGain{D,Z2,P2,
 
 Base.inv(f::ZeroPoleGain{D}) where {D} = ZeroPoleGain{D}(f.p, f.z, inv(f.k))
 
+function checked_abs(e::Integer)
```

Comment:
Why not use `Base.Checked.checked_abs` instead of rolling our own?

Reply:
Good guess at the name, then :smile:

Reply:
Heh, just stumbled upon https://github.com/JuliaLang/julia/pull/53503 and indeed, `uabs` would be even better, making these cases just work instead of throwing. Says there "`Base.uabs` is already being used by a few packages". So maybe we can join those in the expectation of `uabs` becoming part of the public API.